### PR TITLE
Remove static methods from the BetterTreeFeature class that are duplicated from the TreeFeature class

### PR DESCRIPTION
**Pull request by haykam821**

### CODE REVIEW

File: src/main/java/me/hydos/lint/world/feature/BetterTreeFeature.java [hunk 1]
```diff
@@ -143,7 +108,7 @@ private boolean generate(ModifiableTestableWorld world, Random random, BlockPos
 		}
 
 		if (blockPos2.getY() >= 1 && blockPos2.getY() + trunkPlacerHeight + 1 <= 256) {
-			if (!isDirtOrGrass(world, blockPos2.down())) {
+			if (!TreeFeature.isDirtOrGrass(world, blockPos2.down())) {
```
Comment (hYdos) [inline src/main/java/me/hydos/lint/world/feature/BetterTreeFeature.java]: did you test this?
we duplicate it because we dont use minecraft dirt or grass

**Review by hYdos — CHANGES_REQUESTED** [review]

File: src/main/java/me/hydos/lint/world/feature/BetterTreeFeature.java (hunk 1 above)
Comment (haykam821) [reply src/main/java/me/hydos/lint/world/feature/BetterTreeFeature.java]: Fixed in https://github.com/hYdos/Lint/pull/69/commits/7734ef28eac79e95aecebc9e58f72658c6720e01

**Comment by valoeghese** [comment]
y are checks failing

**Review by hYdos — APPROVED** [review]

File: src/main/java/me/hydos/lint/world/feature/BetterTreeFeature.java [hunk 2]
```diff
@@ -171,7 +141,7 @@ private int method_29963(TestableWorld testableWorld, int i, BlockPos blockPos,
 			for (int l = -k; l <= k; ++l) {
 				for (int m = -k; m <= k; ++m) {
 					mutable.set(blockPos, l, j, m);
-					if (!canTreeReplace(testableWorld, mutable) || !treeFeatureConfig.ignoreVines && isVine(testableWorld, mutable)) {
+					if (!TreeFeature.canTreeReplace(testableWorld, mutable) || !treeFeatureConfig.ignoreVines && TreeFeature.isVine(testableWorld, mutable)) {
```
Comment (ramidzkh) [inline src/main/java/me/hydos/lint/world/feature/BetterTreeFeature.java:144]: `TreeFeature.isVine` is private [according to the build](https://github.com/hYdos/Lint/pull/69/checks?check_run_id=2302394094#step:5:68)

Comment (haykam821) [reply src/main/java/me/hydos/lint/world/feature/BetterTreeFeature.java:144]: Fixed in accessible    method    net/minecraft/world/gen/feature/TreeFeature    isVine    (Lnet/minecraft/world/TestableWorld;Lnet/minecraft/util/math/BlockPos;)Z

Comment (haykam821) [reply src/main/java/me/hydos/lint/world/feature/BetterTreeFeature.java:144]: Fixed in 0abce5bfc576a42e9f67b0dd05e76f8e6f967c86

**Pull request merged**